### PR TITLE
Comprehensive End Session Test

### PR DESCRIPTION
**Pull request by SDESkowronski**
This test cases is based on Lars' test video: End Session and Edit

### CODE REVIEW

File: cypress/integration/Train/MyNameIs.spec.js [hunk 1]
```diff
@@ -60,9 +60,9 @@ describe('My Name Is - Train Dialog', () => {
       // Bug 1956: Replaced Entity Indicator has been lost when training, can still be seen when editing a training
       // This test will fail once the bug has been fixed.
       // Also uncomment the 3rd line of code.
```
Comment (LarsLiden) [inline cypress/integration/Train/MyNameIs.spec.js]: Does comment need updating?  Seems like you uncommented it?

File: cypress/support/Train.js [hunk 2]
```diff
@@ -261,7 +261,7 @@ export function BranchChatTurn(originalMessage, newMessage, originalIndex = 0) {
 }
 
 export function SelectAndVerifyEachChatTurn(index = 0) {
-  if (index == 0) editDialogModal.GetAllChatTurns()
+  if (index == 0) editDialogModal.CreateAliasForAllChatTurns()
```
Comment (LarsLiden) [inline cypress/support/Train.js]: Optional:  generally I'm not a fan of one line "if" statements.  Our style is to { } everything

File: cypress/integration/Train/MyNameIs.spec.js (hunk 1 above)
Comment (SDESkowronski) [reply cypress/integration/Train/MyNameIs.spec.js]: Yep, I had intended to do that after I ran it on CircleCI as well...just forgot. Gone now.

File: cypress/support/Train.js (hunk 2 above)
Comment (SDESkowronski) [reply cypress/support/Train.js]: Fixed

File: cypress/support/Helpers.js [hunk 3]
```diff
@@ -65,4 +65,9 @@ export function Moment(dateTime) {
   return undefined
 }
 
-
+// This will return the Inner Text of an element without markup nor newline characters.
+// Needed because each browser handles this functionality differently.
+export function InnerText(element)
```
[the remark this answers is not in the document]
Comment (SDESkowronski) [reply cypress/support/Helpers.js]: Agreed. Fixed.

**Pull request merged**